inference/core/workflows/core_steps/visualizations/common/base_colorable.py

```diff
@@ -77,7 +77,7 @@ class ColorableVisualizationManifest(PredictionsVisualizationManifest, ABC):
         Selector(kind=[STRING_KIND]),
     ] = Field(  # type: ignore
         default="DEFAULT",
-        description="Color palette to use for annotations.",
+        description="Select a color palette for the bounding boxes.",
```

Comment:
I believe this property is used for non bounding boxes (ex. polygons). would double check the wording